wp-parsely.css

```diff
@@ -15,7 +15,6 @@
 .parsely-recommended-widget-entry img {
 	float: left;
 	margin-right: 15px;
-	max-width: 50%;
```

Comment:
How does this change fit in?

Reply:
Styles weren't applied correctly in the past and this would result in the image looking too small with the changes in this PR.